Lib/fontTools/feaLib/variableScalar.py

```diff
@@ -1,4 +1,11 @@
 from fontTools.varLib.models import VariationModel, normalizeValue
+from functools import cached_property, lru_cache
+
+# We will often use exactly the same locations (i.e. the font's
+# masters) for a large number of variable scalars. Instead of
+# creating a model for each, let's share the models.
+model_pool = {}
```

Comment:
Maybe find a right time to empty this dict, or cap its size, so it doesn't become a memory leak in a potential scenario of using fontTools in a long-running Python process?

Reply:
maybe make it a class property of VariableScalar and add a class method to clear_cache() so caller can dump it when finished. Otherwise these will be kept alive forever